modules/RTC/RTCUtils.js

```diff
@@ -864,19 +866,15 @@ class RTCUtils extends Listenable {
                     };
                 }
             } else if (RTCBrowserType.isEdge()) {
-                // TODO: Remove when EDGE is fully supported. For now ensure
-                // that, if EDGE is detected, it's just unsupported.
-                if (RTCBrowserType.isEdge()) {
+                // Disable until fully implemented.
+                if (!ENABLE_EDGE) {
                     rejectWithWebRTCNotSupported(
-                        'Microsoft EDGE not yet supported', reject);
+                        'Microsoft Edge not yet supported', reject);
 
                     return;
                 }
 
-                // TODO: Uncomment when done. For now use the Edge native
-                // RTCPeerConnection.
-                // this.peerconnection = ortcRTCPeerConnection;
-                this.peerconnection = RTCPeerConnection;
+                this.peerconnection = ortcRTCPeerConnection;
```

Comment:
Also, I understand your frustration, but please conduct yourself with decorum, this is a public issue tracker.